packages/mantine/package.json

```
@@ -0,0 +1,97 @@
+{
+  "name": "@rjsf/mantine",
+  "version": "6.0.0-alpha.0",
+  "main": "dist/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "description": "Mantine theme, fields and widgets for react-jsonschema-form",
+  "files": [
+    "dist",
+    "lib",
+    "src"
+  ],
+  "engineStrict": false,
+  "engines": {
+    "node": ">=14"
```

we dropped node 14 support
```suggestion
    "node": ">=20"
```